### PR TITLE
Remake "getting started with WDL" tutorial

### PR DESCRIPTION
**Pull request by aofarrel**
* Remove virtually all references to tools
* Link to a new repository that houses a reworked WDL with 1.0 syntax
* Don't talk about the Docker image as much
* Remove quay.io and other tool related stuff
* New headings to make navigation a little easier
* Include how to glob outputs, since that is a very common use case
* Include a mention of WDL 1.0's alternative command section syntax
* Improve alt text
* Cut down on the output from running the workflow on the command line, and updating the remaining part to the current version of Cromwell/the Dockstore CLI
* Clarifications (hopefully...)

### CODE REVIEW

File: docs/getting-started/getting-started-with-wdl.rst [hunk 1]
```diff
@@ -68,48 +62,61 @@ repository:
...
+            author: "Andrew Duncan"
+            email: "andrew@foobar.com"
+            description: "## Bamstats \n This is the Bamstats workflow.\n\n Adding documentation improves clarity."
+        }
     }
 
+
 Let us break it down piece by piece.
 
+.. note:: The very first line represents the version of WDL specification being used, not necessarily the version of the workflow. In this case, that is `version 1.0` of the WDL spec. If this line is not included, the WDL will be parsed as if it were following the draft-2 specification. Note that most 1.0 WDLs and draft-2 WDLs are not interchangable without a few changes, so make you are using the correct version when following along with examples.
```
[the remark this answers is not in the document]
Comment (aofarrel) [reply docs/getting-started/getting-started-with-wdl.rst:101]: ie, cutting out "not necessarily the version of the workflow"?

[the remark this answers is not in the document]
Comment (coverbeck) [reply docs/getting-started/getting-started-with-wdl.rst:101]: Yes, correct. It's not the version of the workflow, it's the WDL specification.

**Review by coverbeck — APPROVED** [review]
Assuming the logo usage is sorted out.

File: docs/getting-started/getting-started-with-wdl.rst [hunk 2]
```diff
@@ -1,8 +1,4 @@
-.. role:: raw-latex(raw)
-   :format: latex
-..
-
-.. note:: This tutorial is a continuation of :doc:`Getting Started With Docker <getting-started-with-docker>`. Please complete that tutorial prior to doing this one.
+.. note:: This document assumes you have basic knowledge of what Docker is. It is recommended, but not required, to complete :doc:`Getting Started With Docker <getting-started-with-docker>` before this tutorial.
```
Comment (wshands) [inline docs/getting-started/getting-started-with-wdl.rst:1]: 'It is recommended, but not required, to complete' -> 'It is recommended, but not required, that you complete'

**Review by wshands — APPROVED** [review]

File: docs/advanced-topics/dockstore-cli/dockstore-cli-faq.rst [hunk 3]
```diff
@@ -9,10 +9,9 @@ For general FAQs not related to the Dockstore CLI, please see :doc:`our main FAQ
 How does launching CWLs with Dockstore CLI compare with cwltool?
 ----------------------------------------------------------------
 
-Under the hood, the Dockstore CLI evokes cwltool to launch CWL workflows. However, it adds additional features. Dockstore CLI has utilities to generate JSON parameter files from
-entries on Dockstore (``dockstore tool convert``).
-
-When launching tools, the Dockstore CLI makes it easy to specify entries
+Under the hood, the Dockstore CLI evokes cwltool to launch CWL workflows. However, it adds additional features. The Dockstore CLI can generate JSON parameter files from
```
[the remark this answers is not in the document]
Comment (avani-k) [reply docs/advanced-topics/dockstore-cli/dockstore-cli-faq.rst:12]: +1

**Review by avani-k — APPROVED** [review]

File: docs/index.rst [hunk 4]
```diff
@@ -82,13 +173,39 @@ going straight to the :doc:`End User Topics <end-user-topics/end-user-topics>` o
    advanced-topics/dockstore-cli/developing-file-provisioning-plugins
    advanced-topics/dockstore-cli/advanced-features
 
+Advanced developer topics
```
[the remark this answers is not in the document]
Comment (aofarrel) [reply docs/index.rst:176]: Thanks for catching this, that's actually the result of a merge conflict. I forgot to remove "introduction" from the RST it's importing on this branch. I'll tidy that up.

**Comment by aofarrel** [comment]
In order to merge this without waiting for clarification on the logo situation, I will adjust index.rst to not use any logos in that section.

**Comment by bethsheets** [comment]
Hey @aofarrel, this looks great! Some things that I thought about while reading through that may be a good next PR?
1) I don't know if we say this idea in each language guide or maybe in a new general `getting started with workflows` guide -- but many users will come from HPCs and may not immediately understand that these examples are not easily runnable on HPCs due to our workflows using Docker. Do we want to provide a brief 'note' section explaining that? Something like "Where can I run workflows? Locally with the Dockstore CLI or in the cloud, but some changes are necessary to run on a workflow on a High Performance Compute Cluster"
2) Are we missing some basic tools people need to know? Like giving an example editor they can write their workflow in.   Do we want to bring up womtool?
3) Do we think the way this document currently links with other essential docs is an easy-to-navigate user path? For example, creating a Dockstore account and using GitHub Apps to register your workflow? Maybe we could provide more explicit links to the next appropriate parts of the learning experience. To me, it doesn't make sense to move from this WDL doc to a new language when clicking 'next'?

**Pull request merged**